xml/cap_depl_stratos.xml

```diff
@@ -230,6 +230,36 @@ gp2scoped       kubernetes.io/aws-ebs   1d
    Use this IP address to create an A record.
   </para>
  </sect1>
+ <sect1 xml:id="sec.cap.stratos-gke">
+  <title>Deploy Stratos on &gke;</title>
+
+  <para>
+   Before deploying Stratos, ensure <literal>uaa</literal> and <literal>scf</literal>
+   have been successfully deployed on &gke; (see <xref linkend="cha.cap.depl-gke"/>).
+  </para>
+
+  <para>
+   Use &helm; to install Stratos:
+  </para>
+<screen>helm install suse/console \
```

Comment:
use entity for command prompt

Reply:
Thanks! Entity added now for all missing instances.